src/Sauce/Sausage/TestCase.php

```diff
@@ -27,10 +27,13 @@ public function setUp()
             $this->setDesiredCapabilities($caps);
         }
         
-        if (null !== getenv('SAUCE_TUNNEL_IDENTIFIER')) {
-            $caps = $this->getDesiredCapabilities();
-            $caps['tunnel-identifier'] = getenv('SAUCE_TUNNEL_IDENTIFIER');
-            $this->setDesiredCapabilities($caps);
+        $tunnelId = getenv('SAUCE_TUNNEL_IDENTIFIER');
+        if (null !== $tunnelId) {
```

Comment:
I think maybe we don't need to be so strict here. What about just:

```php
if ($tunnelId) {
```

I think we're fine as long as we assume tunnel ids can't be "0"